docs/stellar-core_example.cfg

```diff
@@ -29,7 +29,7 @@ HISTOGRAM_WINDOW_SIZE=30
 # This will get written to a lot and will grow as the size of the ledger grows.
 BUCKET_DIR_PATH="buckets"
 
-
+HISTORY_ARCHIVE_RETRY_LIMIT = 10
```

Comment:
Any particular reason to pick 10 here? This changes the retry strategy we currently have implemented. For downloading files from the archives, we use `GetAndUnzipRemoteFileWork` which defaults to BasicWork::RETRY_A_LOT (32), so I think we should keep that as default.

Reply:
Thinking about this a bit more, what do we want to achieve with this config in general? Glancing at the original issue, the discussion was around adding more visibility into download patterns, so I'm trying to understand how operators would use this new config.